translator-files/po/fr/Software-Manual.po

```diff
@@ -241,9 +247,16 @@ msgstr "## Effet réverbération"
 
 #. type: Plain text
 #: wiki/en/Software-Manual.md:68
+#| msgid ""
+#| "Reverb can be applied to one local mono audio channel or to both channels "
+#| "in stereo mode.  The mono channel selection and the reverberation level "
+#| "can be modified. For example, if a microphone signal is fed in to the "
+#| "right audio channel of the sound card and a reverb effect needs to be "
+#| "applied, set the channel selector to the right and move the fader upwards "
+#| "until the desired reverb level is reached."
 msgid ""
-"Reverb can be applied to one local mono audio channel or to both channels in "
-"stereo mode.  The mono channel selection and the reverberation level can be "
+"Adds reverb to your local mono audio channel, or to both channels in stereo "
```

Comment:
Did this get produced automatically?

Reply:
I'm assuming @jujudusud ran po4a-update-templates, which transferred the recent changes to Software-Manual.md to the .po files and updated them. The lines starting with #| are "previous untranslated strings" and will not show up in the final document.